joystick_control/launch/tele_operation.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
+
+    <node pkg="joystick_control" type="ps4_control.py" name="Tele_Operation"
```

Comment:
We are following the best pratices which includes 2 spaces identation and no whitespace between lines. Also the limit of char per line is 120, so it does not need to be splited is this case.